src/main/webapp/src/app/map/map.component.html

```diff
@@ -0,0 +1,6 @@
+<h1>
+  Welcome to Maps!
+</h1>
+<div class="image">
+  <img src="map.jpg"/>
```

Comment:
I figured out why this doesn't display the image. The image needs to be in the assets folder. I created an 'images' folder in the assets folder and added the image there. Then changed the src to "../../assets/images/map.jpg" and it worked. 
Reason why it should be in the assets folder: https://stackoverflow.com/questions/57535672/cant-display-static-image-in-the-same-folder-as-the-component-in-angular